src/libm-tester/gnuabi_compatibility.c

```diff
@@ -222,7 +221,10 @@ DECLARE_SP(fminf, vv);
 DECLARE_SP(fmodf, vv);
 DECLARE_SP(frfrexpf, v);
 DECLARE_SP(hypotf, vv);
+#ifndef ENABLE_AVX
```

Comment:
it would be good to track this issue in github, and mention it here (and below in the other IFDEF).

Reply:
I think the a proper way to resolve this issue is to call the functions with the correct types of arguments.

Reply:
I see, OK. Then, an issue is needed to track the problem. Please also add how you think this can be solved.

Reply:
And it does not need to be resolved.
Another way is to just remove those two functions.